README.md

```diff
@@ -18,6 +20,35 @@ Some programs/protocols only work over UDP. And some networks only allow TCP. Th
   `tcp2udp` continues to accept new incoming TCP connections, and creates a new UDP socket
   for each. So a single `tcp2udp` server can be used to service many `udp2tcp` clients.
 
+## Building
+
+Ensure you have the required dependencies installed on:
+
+### Debian / Ubuntu
+
+Install Rust by following the guides for [Rustup]([url](https://www.rust-lang.org/tools/install))
+
+Install ibc6-dev with APT:
+
+```
+sudo apt install -y libc6-dev
+```
+
+### Fedora
+
+All dependencies can be installed as follows:
+
+```
+sudo dnf install -y cargo rust glibc-static 
+```
+
+Run the script `build-static-bins.sh` to produce the static binary files. These will be outputted to:
+
+```
+$ ls target/x86_64-unknown-linux-gnu/release/
+build  deps  examples  incremental  tcp2udp  tcp2udp.d  udp2tcp  udp2tcp.d
```

Comment:
You only care about a single file here. I'm not going to mark this as a blocker, do whatever fits you. But if the main problem was to make it easy for someone not super into Rust to just produce a `tcp2udp` from this I would word it as:

> Run the script `build-static-bins.sh` to produce the static binaries. The result will be available at `target/x86_64-unknown-linux-gnu/release/tcp2udp`

No one runs this script to get `udp2tcp` so we can ignore talking about that one.